flink-runtime/src/main/java/org/apache/flink/runtime/state/filesystem/FsCheckpointStreamFactory.java

```diff
@@ -301,9 +301,16 @@ public StreamStateHandle closeAndGetHandle() throws IOException {
 					}
 					else {
 						flush();
+
+						long size = -1;
```

Comment:
Stream position should be okay to determine the state size. All instances I checked were accurate there.
Also, given that the size is more informational and should not be relied upon, it should be all the less critical.
As